### PR TITLE
Add FAQ link in footer

### PR DESCRIPTION
**Pull request by qlemaire22**
Solves https://github.com/stormgateworld/web/issues/43

Also removed API from header since it was a duplicate (it is also in the footer).

### CODE REVIEW

File: src/components/layout/Footer.astro [hunk 1]
```diff
@@ -1,25 +1,32 @@
 ---
 import Svg from "@jasikpark/astro-svg-loader"
+
+const navLinks = [
+  { href: "/faq", text: "FAQ" },
+  { href: "/api", text: "API" },
+  { href: "/privacy", text: "Privacy" },
+  { href: "/about", text: "About" },
+  { href: "https://twitter.com/StormgateWorld", text: "Twitter", newWindow: true, twitter: true },
```
Comment (reneklacan) [inline src/components/layout/Footer.astro:9]: optional ‘target’ and ‘icon’ would work better than newWindow/twitter/rss

**Review by reneklacan — CHANGES_REQUESTED** [review]

File: src/components/layout/Footer.astro (hunk 1 above)
Comment (qlemaire22) [reply src/components/layout/Footer.astro:9]: Right now, there is no icon. So I want wondering if this `<i>` was even used?

Comment (reneklacan) [reply src/components/layout/Footer.astro:9]: ah icons are probably broken since astro upgrade, it uses https://tabler.io/icons/icon/brand-twitter-filled

**Comment by qlemaire22** [comment]
Did the change, I am not sure how to fix icons with astro but I guess this can be another PR

**Review by reneklacan — APPROVED** [review]

**Pull request merged**